sim/hunter/talents.go

```diff
@@ -160,3 +162,11 @@ func (hunter *Hunter) mortalShots() float64 {
 func (hunter *Hunter) trapMastery() float64 {
 	return 5 * float64(hunter.Talents.TrapMastery) * core.SpellHitRatingPerHitChance
 }
+
+func (hunter *Hunter) applyEfficiency() {
+	hunter.OnSpellRegistered(func(spell *core.Spell) {
+		if spell.Flags.Matches(SpellFlagShot) || spell.Flags.Matches(SpellFlagStrike) && spell != hunter.KillShot {
```

Comment:
Just because I don't know what the performance is like comparing spells, could you use SpellCode comparisons for Kill Shot?

Reply:
Not sure about using SpellCodes do you mean something like the actionId or creating a new SpellCode specifically for killshot?

Reply:
Ended up creating a new spell code for killshot